OptionItem.h

```diff
@@ -0,0 +1,68 @@
+#ifndef HEADER_OptionItem
+#define HEADER_OptionItem
+/*
+htop - OptionItem.h
+(C) 2004-2011 Hisham H. Muhammad
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include <stdbool.h>
+
+#include "Object.h"
+
+
+enum OptionItemType {
+   OPTION_ITEM_CHECK,
+   OPTION_ITEM_NUMBER,
+};
+
+typedef struct OptionItemClass_ {
+   const ObjectClass super;
+
+   enum OptionItemType kind;
+} OptionItemClass;
+
+#define As_OptionItem(this_)                ((const OptionItemClass*)((this_)->super.klass))
+#define OptionItem_kind(this_)              As_OptionItem(this_)->kind
+
+typedef struct OptionItem_ {
+   Object super;
+} OptionItem;
+
+typedef struct CheckItem_ {
+   OptionItem super;
+
+   char* text;
```

Comment:
Put `text` in `OptionItem` as common field?

Reply:
changed